tests/integration/RequestBuilder/EventsRequestBuilderTest.php

```diff
@@ -6,13 +6,29 @@
 use Lmc\Matej\IntegrationTests\IntegrationTestCase;
 use Lmc\Matej\Model\Command\Interaction;
 use Lmc\Matej\Model\Command\ItemProperty;
+use Lmc\Matej\Model\Command\ItemPropertySetup;
 use Lmc\Matej\Model\Command\UserMerge;
+use Lmc\Matej\RequestBuilder\ItemPropertiesSetupRequestBuilder;
 
 /**
  * @covers \Lmc\Matej\RequestBuilder\EventsRequestBuilder
  */
 class EventsRequestBuilderTest extends IntegrationTestCase
 {
+    protected function setup(): void
```

Comment:
This should be setUp, and I'm wondering why this was not detected by codestyle check, there should be one exactly for this 🤔